### ISSUE TITLE
ClassCastException on a click with ClickOptions

### ISSUE THREAD

**Issue by wasder** [selenide/selenide-appium#155]
## The problem
There is a ClassCastException when I  try to click an element with offset using ClickOptions. 

## Details

Page Object contains:
```Java
    @iOSXCUITFindBy(id = "someId")
    private SelenideElement element;

    public SomePage clickElement() {
        element.click(ClickOptions.withOffset(0, 10));
        return this;
    }
```
When I invoke `clickElement()` in a test, it throws this exception:

```Java
class com.codeborne.selenide.ClickOptions cannot be cast to class com.codeborne.selenide.appium.AppiumClickOptions (com.codeborne.selenide.ClickOptions and com.codeborne.selenide.appium.AppiumClickOptions are in unnamed module of loader 'app')
java.lang.ClassCastException: class com.codeborne.selenide.ClickOptions cannot be cast to class com.codeborne.selenide.appium.AppiumClickOptions (com.codeborne.selenide.ClickOptions and com.codeborne.selenide.appium.AppiumClickOptions are in unnamed module of loader 'app')
	at com.codeborne.selenide.appium.commands.AppiumClick.execute(AppiumClick.java:44)
	at com.codeborne.selenide.appium.commands.AppiumClick.execute(AppiumClick.java:29)
	at com.codeborne.selenide.commands.Commands.execute(Commands.java:159)
	at com.codeborne.selenide.impl.SelenideElementProxy.dispatchAndRetry(SelenideElementProxy.java:132)
	at com.codeborne.selenide.impl.SelenideElementProxy.invoke(SelenideElementProxy.java:84)
	at jdk.proxy3/jdk.proxy3.$Proxy50.click(Unknown Source)

```

Driver is configured using WebDriverProvider:
```Java
 Configuration.browser = IOSDriverProvider.class.getName();
```

```Java
 
 public class IOSDriverProvider implements WebDriverProvider {

    static EnvConfig cfg = ConfigCache.getOrCreate(EnvConfig.class);

    @CheckReturnValue
    @Nonnull
    @Override
    public WebDriver createDriver(@Nonnull Capabilities capabilities) {
        XCUITestOptions options = new XCUITestOptions();
        options = options.merge(capabilities);
        options.setDeviceName(cfg.IOS_DEVICE());
        options.setApp(cfg.IOS_APP_PATH());
        options.setPlatformVersion(cfg.IOS_VERSION());
        options.setNewCommandTimeout(Duration.ofSeconds(90));
        options.setFullReset(false);
        options.setShouldTerminateApp(false);
        options.setForceAppLaunch(true);
        try {
            IOSDriver iosDriver = new IOSDriver(new URL("http://127.0.0.1:4723"), options);
            iosDriver.setSetting(WAIT_FOR_IDLE_TIMEOUT, 1);//value in seconds
            iosDriver.executeScript("mobile:enrollBiometric", ImmutableMap.of("isEnabled", true));
            return iosDriver;
        } catch (MalformedURLException e) {
            throw new RuntimeException(e);
        }
    }
}

 
```

## Tell us about your environment
* **Selenide Version**:  'com.codeborne:selenide:6.11.2'
* **Selenide-Appium Version**:  'com.codeborne:selenide-appium:2.7.0'

**Comment by wasder** [comment]
And only after creating the issue I read the Readme and found that I should use `click(tapWithOffset(0, 10));` which works as expected. Sorry for bothering.

**Comment by asolntsev** [comment]
@wasder Thank you for reporting this issue. I rather think it's a bug, and I would like it to be fixed. 
I mean, any `click` that is compilable should also work well.

Hope to fix it in next release.

**Comment by asolntsev** [comment]
Fixed in https://github.com/selenide/selenide-appium/pull/165, will be released as 2.8.0
